doc/source/examples/multiphysics/jurins-law/jurins-law.rst

```diff
@@ -0,0 +1,307 @@
+|                                                                                                                   |
+|     Height difference evolution for different angles of contact (>90° and <90°) with respect to time.             |
+|                                                                                                                   |
++-------------------------------------------------------------------------------------------------------------------+
+
+Furthermore, by visualizing the pressure fields in the vicinity of the meniscus at the end of the simulation, we observe in the following figure that they correspond well qualitatively to the overpressures or depressions predicted by Young-Laplace's law. We conclude that the contact angle condition is adequately coupled with the Navier-Stokes equations.
+
++-------------------------------------------------------------------------------------------------------------------+
+|  .. figure:: images/pressure_difference.png                                                                       |
+|     :alt: Representation of the pressure field at the last time-step of the simulation (t = 0.498212 s). The      |
+|      pressure gradient at the vicinity of the interface corresponds to that expected by the Young-Laplace         |
+|       equation,                                                                                                   |
+|      with an overpressure at positive curvature interfaces and depressions at negative curvature interfaces.      |
+|     :align: center                                                                                                |
+|     :name: Pressure field at the end of the simulation                                                            |
+|                                                                                                                   |
+|     Pressure fields at the end of the simulation for different angles of contact.                                 |
+|                                                                                                                   |
++-------------------------------------------------------------------------------------------------------------------+
+
+---------------------------
+Possibilities for Extension
+---------------------------
+
+- **Going 3D**: the mesh can be extruded into the third dimension and there is an adaptation of the Jurin's law in three dimensions. Some results are available in the literature for comparison (see Lovrić et al. [#lovric2019]_)
+
+- **Investigate the effect of a no-slip boundary condition**: instead of the slip boundary condition imposed on the inner face of the wall, we could try to use a no-slip boundary condition. This situation would be closer to a real capillary rise experiment. We expect to observe a different transitory state with this new boundary condition.
```

Comment:
```suggestion
- **Investigate the effect of a no-slip boundary condition**: instead of the slip boundary condition imposed on the inner face of the wall, we could try to use a no-slip boundary condition. This situation would be closer to a real capillary rise experiment. We expect to observe a different transient state with this new boundary condition.
```